README.md

```diff
@@ -210,6 +211,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/ZeeshanMukhtar1"><img src="https://avatars.githubusercontent.com/u/91063160?v=4?s=100" width="100px;" alt="Zeeshan Mukhtar"/><br /><sub><b>Zeeshan Mukhtar</b></sub></a><br /><a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=ZeeshanMukhtar1" title="Code">💻</a> <a href="#content-ZeeshanMukhtar1" title="Content">🖋</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=ZeeshanMukhtar1" title="Documentation">📖</a> <a href="#example-ZeeshanMukhtar1" title="Examples">💡</a> <a href="#ideas-ZeeshanMukhtar1" title="Ideas, Planning, & Feedback">🤔</a> <a href="https://github.com/TejasQ/opensauced-guestbook/pulls?q=is%3Apr+reviewed-by%3AZeeshanMukhtar1" title="Reviewed Pull Requests">👀</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=ZeeshanMukhtar1" title="Tests">⚠️</a> <a href="#translation-ZeeshanMukhtar1" title="Translation">🌍</a> <a href="#tutorial-ZeeshanMukhtar1" title="Tutorials">✅</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/codewithjazzy"><img src="https://avatars.githubusercontent.com/u/105261487?v=4?s=100" width="100px;" alt="Jasmine"/><br /><sub><b>Jasmine</b></sub></a><br /><a href="#tutorial-codewithjazzy" title="Tutorials">✅</a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/VirginieLemaire"><img src="https://avatars.githubusercontent.com/u/82816247?v=4?s=100" width="100px;" alt="Virginie"/><br /><sub><b>Virginie</b></sub></a><br /><a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=VirginieLemaire" title="Code">💻</a> <a href="#data-VirginieLemaire" title="Data">🔣</a> <a href="https://github.com/TejasQ/opensauced-guestbook/commits?author=VirginieLemaire" title="Documentation">📖</a> <a href="#ideas-VirginieLemaire" title="Ideas, Planning, & Feedback">🤔</a></td>
```

Comment:
Let's remove this line.
It will be regenerated after you run `npm run contributors:generate` command.

Reply:
ok ;)